pkg/pillar/cmd/volumemgr/handlevolumeref.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2020 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package volumemgr
+
+import (
+	"github.com/lf-edge/eve/pkg/pillar/types"
+	log "github.com/sirupsen/logrus"
+)
+
+func handleVolumeRefModify(ctxArg interface{}, key string,
+	configArg interface{}) {
+
+	log.Infof("handleVolumeRefModify(%s)", key)
+	config := configArg.(types.VolumeRefConfig)
+	ctx := ctxArg.(*volumemgrContext)
+	vs := lookupVolumeStatus(ctx, config.VolumeKey())
+	if vs != nil {
+		vs.RefCount++
+		publishVolumeStatus(ctx, vs)
+	}
```

Comment:
If zedmanager is processing a volumeRefList entry before volumemgr has received and processed the associated VolumeConfig, then you're refcounts will be permanently out of whack. A subsequent handleVolumeRefDelete will decrement something which was not incremented.

I think zedmanager must have a way to track this e.g., by having a bool in VolumeRefStatus which called MissingVolume, and then go back and look for such to update the refcount in the VolumeStatus. But I don't even see management of a VolumeRefStatus in here.